### PR TITLE
Pm mck feature/add sqs

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

File: lwt/runtime.ml [hunk 1]
```diff
@@ -46,11 +46,12 @@ let run_request
     (inp : M.input)
   =
   let meth, uri, headers =
-    Aws.Signing.sign_request ~access_key ~secret_key ~service:M.service ~region (M.to_http inp)
+    Aws.Signing.sign_request ~access_key ~secret_key ~service:M.service ~region (M.to_http M.service region inp)
   in
   let open Cohttp in
   let headers = Header.of_list headers in
   Lwt.catch (fun () ->
+    Printf.printf "HTTP %s: %s" (Aws.Request.string_of_meth meth) (Uri.to_string uri);
```
Comment (pm-mck) [inline lwt/runtime.ml]: @tmcgilchrist do you want the logging here?

**Comment by pm-mck** [comment]
@tmcgilchrist are you ok with the decisions made in the PR? I think it should scale to all AWS services, but adding atdgen and a new executable are decisions I'd like your input on.

File: lwt/runtime.ml (hunk 1 above)
Comment (tmcgilchrist) [reply lwt/runtime.ml]: Really no, the `async` runtime does some logging and I added this mostly for debug reasons.
What it should do is accept a `verbose` option and log based off that. Larger problem and more effort, for another PR.

**Comment by tmcgilchrist** [comment]
I'm ok with adding `atdgen` it's common enough to depend on but having a single entry point for code generation would be better.  Don't mind how that gets done (either here or another branch)
@pm-mck

I started branching off this work to start generating other services I need. So far it looks good. 
The project is quite light on for CI tests that hit AWS proper so I have been running things locally.

File: lwt/runtime.ml (hunk 1 above)
Comment (pm-mck) [reply lwt/runtime.ml]: Agreed

**Comment by pm-mck** [comment]
I'm happy to start looking at how to add localstack to help improve the state of tests, do you think that should be treated as another issue? This pr is already massive.

**Comment by tmcgilchrist** [comment]
Yes, keen to merge this and keep working on other improvements.

**Pull request merged**